### DISCUSSION TITLE
How to determine crossfeeding between two species?

### DISCUSSION THREAD

**Discussion by hannahschrenk** [micom-dev/micom#151] [Q&A] [answered]
We would like to extract the exact nutrient exchange (which nutrients, amount of them) between two species, however, we did not find a clear indication on how to do that. We have tried with elasticity measures, and by comparing influx and outflux, but perhaps there is a more straightforward way to gain this information? Any insight into this would be greatly appreciated. 
Thank you for your help!

**Comment by cdiener** [comment, marked as answer]
If you use the `grow` workflow that information is contained in `GrowthResults.exchanges` which maps out which metabolites are produced by each microbe and which ones are consumed. If species A produces M and species B consumes it that would be a trophic interaction. If both consume it they might compete for it. A new interaction module is currently in the works, and it will give you that directly along with other measures such as total exchanges mass/carbon/nitrogen flux etc.

**Reply by hannahschrenk** [reply]
Thank you very much!
